.github/workflows/test.yml

```
@@ -7,20 +7,46 @@ jobs:
       - name: Generate application key
         run: php artisan key:generate
-      - name: Run tests
+
+      # 7) Run your test suite
+      - name: Run PHPUnit tests
         run: php artisan test
+
+      # 8) Parallel tests in random order
       - name: Run parallel tests randomly
-        run: php artisan test --parallel --processes=4 --order-by=random
+        run: php artisan test --parallel --processes=4 --order-by=random
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)